experimental/cache_size.cc

```diff
@@ -0,0 +1,70 @@
+#include <stdlib.h>
+#include <time.h>
+
+#include <algorithm>
+#include <iostream>
+#include <memory>
+#include <random>
+
+#include "../demos/asm/measurereadlatency.h"
+#include "../demos/instr.h"
+#include "../demos/utils.h"
+
+// Determines the maximum latency of reading elements of a vector of size "sz",
+// which is given as input.
+// It reads each element of the vector and measures the time
+// using MeasureReadLatency(). Later this number can be used to determine cache
+// misses and hits
+
```

Comment:
Super ULTRA nit: delete the blank line. (Function comments are not supposed to have whitespace between them and the function declaration.)